### PR TITLE
[ID] Add `Client/Interface/Chat_console/Highlight`

### PR DESCRIPTION
**Pull request by daud-ahn**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

**Comment by daud-ahn** [comment]
The image is still in English, can't get a good screenshot at the moment.

File: wiki/Client/Interface/Chat_console/Highlight/id.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+tags:
+  - mention
+  - hilight
+  - HL
+  - ping
+---
+
+# Highlight
+
+**Highlight** (biasanya disingkat **HL**, kadang-kadang disebut **mention** atau **ping**) adalah cara untuk menarik perhatian seseorang ke chat jika mereka sedang online. Secara default, highlight dipicu ketika nama pengguna pemain ditulis dalam chat. Daftar kata yang disorot dapat disesuaikan di [pilihan chat dalam game](/wiki/Client/Options#in-game-chat).
```
Comment (mousesalt) [inline wiki/Client/Interface/Chat_console/Highlight/id.md:11]:
```suggestion
**Highlight** (biasanya disingkat **HL**, kadang-kadang disebut **mention** atau **ping**) adalah salah satu cara untuk menarik perhatian seseorang ke dalam chat jika mereka sedang online. Secara umum, highlight aktif ketika nama pengguna ditulis atau disebutkan ke dalam chat. Daftar kata yang di-highlight dapat disesuaikan di dalam klien osu!, bagian [in-game chat options](/wiki/Client/Options#in-game-chat).
```

File: wiki/Client/Interface/Chat_console/Highlight/id.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+---
+tags:
+  - mention
+  - hilight
```
Comment (mousesalt) [inline wiki/Client/Interface/Chat_console/Highlight/id.md:4]:
```suggestion
  - highlight
```

File: wiki/Client/Interface/Chat_console/Highlight/id.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  - HL
+  - ping
+---
+
+# Highlight
+
+**Highlight** (biasanya disingkat **HL**, kadang-kadang disebut **mention** atau **ping**) adalah cara untuk menarik perhatian seseorang ke chat jika mereka sedang online. Secara default, highlight dipicu ketika nama pengguna pemain ditulis dalam chat. Daftar kata yang disorot dapat disesuaikan di [pilihan chat dalam game](/wiki/Client/Options#in-game-chat).
+
+Anda dapat mengabaikan sorotan dari pengguna tertentu dengan menambahkan nama penggunanya ke [daftar pengabaian](/wiki/Client/Options/Ignore_list) dengan `@h` ditambahkan ke nama pengguna tersebut.
```
Comment (mousesalt) [inline wiki/Client/Interface/Chat_console/Highlight/id.md:13]:
```suggestion
Untuk dapat mengabaikan highlight dari pengguna tertentu, tambahkan nama pengguna ke dalam [daftar pengabaian](/wiki/Client/Options/Ignore_list) dengan `@h` sebagai sisipan.
```

File: wiki/Client/Interface/Chat_console/Highlight/id.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+## Penampilan
```
Comment (mousesalt) [inline wiki/Client/Interface/Chat_console/Highlight/id.md:15]:
```suggestion
## Tampilan
```

File: wiki/Client/Interface/Chat_console/Highlight/id.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+::: Infobox
+![](img/hl-notification.png "Contoh notifikasi highlight")
```
Comment (mousesalt) [inline wiki/Client/Interface/Chat_console/Highlight/id.md:18]:
```suggestion
![](img/hl-notification.png "Sebuah contoh notifikasi dari highlight")
```

File: wiki/Client/Interface/Chat_console/Highlight/id.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+:::
+
+Ketika pesan chat memicu highlight, nama pengguna pengirim pada baris tersebut menjadi hijau, dan pesan ditambahkan ke tab chat `#highlight`. Selain itu, jika window osu! tidak aktif, ikonnya akan berkedip di taskbar.
```
Comment (mousesalt) [inline wiki/Client/Interface/Chat_console/Highlight/id.md:21]:
```suggestion
Ketika sebuah pesan mengaktifkan highlight, nama pengguna pengirim pada baris tersebut akan menjadi hijau, dan pesan ditambahkan ke tab chat `#highlight`. Selain itu, jika jendela osu! tidak aktif, ikon pada taskbar akan berkedip.
```

**Review by mousesalt — CHANGES_REQUESTED** [review]

File: wiki/Client/Interface/Chat_console/Highlight/id.md (hunk 2 above)
Comment (daud-ahn) [reply wiki/Client/Interface/Chat_console/Highlight/id.md]: I should've read the reply before committing. I will revert this along with the unresolved suggestions.

File: wiki/Client/Interface/Chat_console/Highlight/id.md (hunk 1 above)
Comment (mousesalt) [reply wiki/Client/Interface/Chat_console/Highlight/id.md:11]: pilihan apapun, kamu tentukan.
bisa digunakan selama kategori penyebutan masuk akal. tidak selamanya translasi wajib sempurna dengan versi original.

**Review by TPGPL — APPROVED** [review]

**Pull request merged**